flare/examples/distributional_rl/c51_example.py

```diff
@@ -0,0 +1,75 @@
+import torch.nn as nn
```

Comment:
Maybe consider integrating all the similar examples into one single file and let the user specify the algorithm/model. Currently too much repeated code.

Reply:
Example files are merged.